externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/schema/RowSet.scala

```diff
@@ -151,13 +156,8 @@ object RowSet {
         while (i < rowSize) {
           val row = rows(i)
           nulls.set(i, row.isNullAt(ordinal))
-          val value =
-            if (row.isNullAt(ordinal)) {
-              ""
-            } else {
-              toHiveString((row.get(ordinal), typ), timeZone)
-            }
-          values.add(value)
+          values.add(
+            HiveResult.toHiveString((row.get(ordinal), typ), false, getTimeFormatters(timeZone)))
```

Comment:
nested = true

Reply:
refer to `org.apache.spark.sql.hive.thriftserver.RowSetUtils#toTColumn()`

Reply:
As offline discussed, `nested` should be `false` here, but it does not affect the output.